src/bundle/Resources/views/content/relation.html.twig

```diff
@@ -1,12 +1,16 @@
 <td>
-    <a {% if not content.contentInfo.trashed %}
-        href="{{ path('_ezpublishLocation', { 'locationId': content.contentInfo.mainLocationId } ) }}"
-    {% endif %}>
+    {% if content.contentInfo.mainLocationId is not null %}
+        <a {% if not content.contentInfo.trashed %}
```

Comment:
Isn't main location null when content is in trash? _(in trash location data is moved to another table, so it's no longer in the main tree)_
_Aslo, currently when content is in trash this code will generate empty a tag it seems_


If so, maybe rather:
```
    {% if content.contentInfo.mainLocationId is not null %}
        <a href="{{ path('_ezpublishLocation', { 'locationId': content.contentInfo.mainLocationId } ) }}">
            {{ content.getName() }}
        </a>
    {% else %}
        {{ content.getName() }}
        {% if content.contentInfo.trashed %}
            <em>({{ 'content.in_trash'|trans(domain='content')|desc('In Trash') }})</em>
        {% endif %}
    {% endif %}
```

Reply:
@andrerom yeah, that's a much better way to do this, resolved in https://github.com/ezsystems/ezplatform-admin-ui/pull/1353/commits/e0a9a855cf4de0faa41d371139cd3da64ee5da1b